src/core/config.go

```diff
@@ -442,6 +443,7 @@ type Configuration struct {
 		GoPath           string `help:"If set, will set the GOPATH environment variable appropriately during build actions." var:"GOPATH"`
 		ImportPath       string `help:"Sets the default Go import path at the root of this repository.\nFor example, in the Please repo, we might set it to github.com/thought-machine/please to allow imports from that package within the repo." var:"GO_IMPORT_PATH"`
 		CgoCCTool        string `help:"Sets the location of CC while building cgo_library and cgo_test rules. Defaults to gcc" var:"CGO_CC_TOOL"`
+		CgoEnabled       string `help:"Sets the CGO_ENABLED which controls whether the cgo build flag is set. Defaults to 0 (off)" var:"CGO_ENABLED"`
```

Comment:
Is defaulting it to off correct? Go defaults it to 1 doesn't it?

Reply:
I don't think so. For example, if you don't set it manually, `go install std` doesn't compile `runtime/cgo`. `go build` sets it and compile `runtime/cgo` etc. on the fly like how it compiles the standard library when you cross compile or do `-race`. It's hard for us to do that with our approach though.